server/session/handler_item_stack_request.go

```diff
@@ -485,3 +556,184 @@ func (h *ItemStackRequestHandler) reject(id int32, s *Session) {
 	}
 	h.changes = map[byte]map[byte]changeInfo{}
 }
+
+// inputMapFromInputs makes an input map from inputs.
```

Comment:
Not clear by this documentation exactly what this is for, might want to expand this documentation a bit.